packages/components/src/mobile/keyboard-aware-flat-list/index.android.js

```diff
@@ -8,9 +8,16 @@ import { FlatList } from 'react-native';
  */
 import KeyboardAvoidingView from '../keyboard-avoiding-view';
 
-export const KeyboardAwareFlatList = ( props ) => (
+export const KeyboardAwareFlatList = ( { containerStyle, ...props } ) => (
 	<KeyboardAvoidingView style={ { flex: 1 } }>
-		<FlatList { ...props } />
+		<FlatList
+			scrollEnabled={ containerStyle ? false : true }
```

Comment:
Why do we want to block the scroll when we pass the `containerStyle` ?

Reply:
I wrote some comments above that `containerStyle` maybe isn't the most proper name.

However at this moment I'm passing containerStyle only for Columns block (which holds Column block inside). This style object change the flex direction to 'row' to achieve Column wrapping depends on screen size and number of Column added to Columns block. Without casting `scrollEnabled` to false the UI allows to scroll container to left and right.

Reply:
Yeah, what i meant - at this moment You use that prop only for column block and it's fine, but i can imagine that in the future we would need to pass `containerStyle` as well but w/o blocking the scroll. I think it is better to pass more props to control that instead of relying only on `containerStyle` . WDYT?

Reply:
Agree that we shouldn't based on `containerStyle` to set scrollEnabled. No sure either if we want to pass it from InnerBlock. Maybe we are able to find better solution